Dashboard.Storage/Azure/BuildUtil.cs

```diff
@@ -71,6 +71,30 @@ public List<BuildFailureEntity> GetTestCaseFailures(DateTimeOffset startDate, st
             return _buildFailureDateTable.ExecuteQuery(query).ToList();
         }
 
+        public List<string> GetViewNames(DateTimeOffset startDate)
```

Comment:
I think this approach is fine now.  Going forward though I wonder if we should have a separate table that tracks view names.  

For a given date we have thousands of rows but only about 10-20 view names.  That means our select here is bringing down thousands of pieces of data that we don't use.  May behoove us to build a parallel table that just has the data that we need.  I'll file a bug to track that work.